src/dir.targets

```diff
@@ -172,4 +172,27 @@
 
     <Error Condition="'$(TestRunExitCode)' != '0'" Text="One or more tests failed while running tests from '$(MSBuildProjectName)' please check log for details!" />
   </Target>
+
+  <!-- Generate coverage for individual projects. -->
+  <PropertyGroup>
+    <ProjectInput>$(MSBuildProjectName).dll.coverage.xml</ProjectInput>
```

Comment:
I would specify the full path here and also move this up higher to share it with CoverageCommandLine so that everything stays in sync.